src/cityreader/cityreader.py

```diff
@@ -16,26 +29,32 @@
 # should not be loaded into a City object.
 cities = []
 
+
 def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
+  # For each city record, create a new City instance and add it to the
   # `cities` list
-    
+    with open('cities.csv', 'r') as cities_csv:
+        csv_reader = csv.reader(cities_csv)
+        next(csv_reader)
+        for line in csv_reader:
+            cities.append(City((line[0]), float(line[3]), float(line[4])))
     return cities
```

Comment:
Excellent usage of `next()` to skip the header of the csv and great choice of `line` as your iterative variable.